### DISCUSSION TITLE
After deployment of statamic update html templates use old statamic vendor assets

### DISCUSSION THREAD

**Discussion by toxsick** [statamic/cms#10275] [Troubleshooting] [answered]
Hey guys,

I've run into an issue with my test deployment a few times now that happens **when a new statamic version is deployed**. After the upgrade cp pages html loads old (pre-update) CSS and JS files (e.g. app-08f252f3.js instead of app-3423cfef.js). The new assets are definitely there in /vendor/statamic/cp/build/assets/, but for some reason, the HTML is stuck on the old ones.

I also tried clearing all kinds of caches, re-installing, etc, but the result is the same.

Screenshot for reference:

![Monosnap Log in ‹ Statamic 2024-06-06 09-52-35 2](https://github.com/statamic/cms/assets/951466/7f47913c-5f9d-4f90-b0fb-05e10660afea)

![Monosnap Log in ‹ Statamic 2024-06-06 09-53-54 2](https://github.com/statamic/cms/assets/951466/202ff197-9635-4f3e-81a5-ba49b26a00f3)

![Folder-2024-06-06 09-55-03 2](https://github.com/statamic/cms/assets/951466/e05bf790-c0c0-457a-a416-5f66bc80d49e)

Any ideas on how to get the HTML to reference the updated assets? I am relatively new to the laravel world, perhaps this is something super obvious, but I tried to find similar existing issues and found nothing so far.

I can into more detail on how the deployment works (with deployer), but perhaps you guys know the answer right away.

Help would be much appreciated. Thanks!

**Comment by edalzell** [comment]
This is usually browser cache, or if you're using a proxy, the proxy cache, clear those

**Reply by toxsick** [reply]
Hey @edalzell, thanks for that info, I will check with the provider if there is a cache layer that I'm not aware of. I will add details here for someone facing similar issues in the future.

**Reply by toxsick** [reply, marked as answer]
OK for anyone having similar issues in the future, it was the **php OPcache**^^ Very obvious to a php person I guess. I think it depends on your hoster how you can clear it, but after clearing it the assets are referenced correctly.
